### PR TITLE
bug fix: tooltip cut on small screen

### PR DESCRIPTION
**Pull request by noyagasi**
Should solve #2180 , but please check it before you approve it because I haven't yet.. (couldn't install Amplication on my Windows PC)

### CODE REVIEW

**Comment by yuval-hazaz** [comment]
@noyagasi thanks for the PR!
It looks shart 🤩

We would love to talk to better understand what was the issue running on your windows machine

I see that you joined our Discord server, I will ping you there

File: packages/amplication-design-system/src/components/UserAndTime/UserAndTime.tsx [hunk 1]
```diff
@@ -8,23 +8,23 @@ export type Props = {
...
 
-const DIRECTION = "n";
-
 export function UserAndTime({ account, time }: Props) {
+  const [tooltipDirection, setTooltipDiretion] = useState("s");
   const { firstName, lastName } = account || {};
   const formattedTime = useMemo(() => {
     return formatTimeToNow(time);
   }, [time]);
+  const changeTooltipDirection = (pageY: number) => setTooltipDiretion(pageY < 100 ? "s" : "n");
```
Comment (tupe12334) [inline packages/amplication-design-system/src/components/UserAndTime/UserAndTime.tsx:17]:
```suggestion
  const changeTooltipDirection = (pageY: number) => setTooltipDirection(pageY < 100 ? "s" : "n");
```

File: packages/amplication-design-system/src/components/UserAndTime/UserAndTime.tsx [hunk 2]
```diff
@@ -8,23 +8,23 @@ export type Props = {
   time: Date;
 };
 
-const DIRECTION = "n";
-
 export function UserAndTime({ account, time }: Props) {
+  const [tooltipDirection, setTooltipDiretion] = useState("s");
```
Comment (tupe12334) [inline packages/amplication-design-system/src/components/UserAndTime/UserAndTime.tsx:12]:
```suggestion
  const [tooltipDirection, setTooltipDirection] = useState("s");
```

**Comment by yuval-hazaz** [comment]
I merge this PR into my other PR to fix the missing types

**Comment by yuval-hazaz** [comment]
@all-contributors please add @noyagasi for code and bug fix

**Pull request merged**